docker/template.dockerfile

```diff
@@ -242,6 +243,15 @@ ENV AMDINFER_ENABLE_MIGRAPHX=ON
 
 $[INSTALL_MIGRAPHX]
 
+FROM migraphx_installer_${ENABLE_MIGRAPHX} AS rocal_installer_no
```

Comment:
not sure why ENABLE_MIGRAPHX is used to control rocAL?

Reply:
It's a chain in the dockerfile so depending on what you've enabled, you add in components on top of previous ones. So here, rocal is continuing from one of two migraphx stages, the exact one depending on if migraphx was enabled or not.